test/gluehbirne_web/live/brainstorming_live_test.exs

```diff
@@ -34,4 +34,18 @@ defmodule MindwendelWeb.BrainstormingLiveTest do
       assert html =~ "Anonym"
     end
   end
+
+  describe "Show - Not found" do
+    test "because of non-existing record", %{conn: conn} do
```

Comment:
i don't understand these test descriptions. What do you want to test? The naming is not clear to me. Same for the others. I mean, just read it aloud: "test because of non-existing record" ...

Reply:
Removed